recipe/meta.yaml

```
@@ -1,16 +1,14 @@
-{% set version = "3.7.1" %}
+{% set version = "3.8.4" %}
 
 package:
   name: singularity
   version: {{ version }}
 
 source:
   - folder: src/github.com/sylabs/singularity
-    url: https://github.com/sylabs/singularity/releases/download/v{{ version }}/singularity-{{ version }}.tar.gz
-    sha256: 82d2c65063560195ec34551931be3c325b95e8e2009e92755fd7daad346e083c
+    url: https://github.com/sylabs/singularity/releases/download/v{{ version }}/singularity-ce-{{ version }}.tar.gz
```

This should be changed to `hpcng` rather than `sylabs`, detaills of what has caused there to be two projects are [here](https://sylabs.io/2021/05/singularity-community-edition) but in summary `hpcng/singularity` is the original repo and `sylabs/singularity` is now a fork named SingularityCE.
```suggestion
    url: https://github.com/hpcng/singularity/releases/download/v{{ version }}/singularity-ce-{{ version }}.tar.gz
```